VideoWeb/VideoWeb/ClientApp/src/app/shared/select-media-devices/select-media-devices.component.scss

```diff
@@ -26,11 +26,14 @@
 .modal-content {
   background-color: #fefefe;
   margin: 5% auto;
-  /* 15% from the top and centered */
-  padding: 20px;
   border: 1px solid #888;
-  width: 50%;
-  /* Could be more or less, depending on screen size */
+  padding: 20px;
+  @media only screen and (min-width: 1024px) {
+    width: 50%;
+  }
+  @media only screen and (min-width: 280px) and (max-width: 1024px) {
+    width: 82%;
+  }
```

Comment:
Having the min-width of one query overlap with the max-width of another can make this hard to maintain. Make the max-width 1023px.

Reply:
Actually we have been sticking to em for media queries so also convert these to em.

Reply:
Please refer to this as to why em is preferred

https://zellwk.com/blog/media-query-units/

Pixels can causes issues if the user has changed their default font size. People who have poor sight might do this for example.